v-next/hardhat/README.md

```diff
@@ -1,3 +1,33 @@
-# Hardhat `v-next`
+# Hardhat 3 Alpha
 
-To try this out see `../example-project`
+Hardhat 3 is a major overhaul with exciting new features:
+
+- 🧪 **Solidity tests** as a first-class testing option
+- 🌐 **Multichain support** for today's rollup-centric world
+- ⚡ **Rust-powered runtime** for faster execution
+- 🧱 **Revamped build system** with full npm compatibility and build profiles
+- 🚀 **Hardhat Ignition** for streamlined contract deployments
+
+It's currently in alpha state, but you can try it out and give us feedback!
+
+## Getting started
+
+To install Hardhat 3, run the following commands in an empty directory:
+
+```bash
+npm init -y
+npm install --save-dev hardhat@next
+npx hardhat --init
+```
+
+This will take you through an interactive setup process to get started using Hardhat 3.
+
+## Learn more
```

Comment:
How about we also let people know where the code for the Hardhat 3 is? As it's not that obvious that it's not in `packages`.

Reply:
We'll change that in a few days